<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>634</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>696</width>
    <height>634</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>slowhttptest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>256</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="serveraddress_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Server Address: </string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output_tb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>681</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="serveraddr_ledit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>http://127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="port_label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_ledit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>80</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>341</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>General Options</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="num_conn_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Connection number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="interval_time_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Interval seconds :</string>
     </property>
    </widget>
    <widget class="QLabel" name="rate_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rate :</string>
     </property>
    </widget>
    <widget class="QLabel" name="test_time_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Test Time(s) :</string>
     </property>
    </widget>
    <widget class="QLabel" name="len_content_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Content length :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="num_conn_le">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="interval_time_le">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="test_time_le">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>240</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rate_le">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="content_len_le">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>4096</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="serveraddress_label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>491</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="testmode_combox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Slow Header(default)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Slow Header(default)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Slow Body</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Range Attack</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Slow Read</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Test Modes:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="report_checkbox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Need Report?</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="actionAbout_me"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionShow_report"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionProbe_proxy_options"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionProbe_proxy_options">
   <property name="text">
    <string>probe/proxy options</string>
   </property>
  </action>
  <action name="actionAbout_me">
   <property name="text">
    <string>about me!</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>help</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionShow_report">
   <property name="text">
    <string>Show report</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
